scala/scala.bzl

```diff
@@ -327,6 +327,17 @@ scala_test = rule(
   test=True,
 )
 
+def scala_version():
+  """return the scala version for use in maven coordinates"""
+  return "2.11"
+
+def scala_mvn_artifact(artifact):
```

Comment:
I agree that such stringly typed code is not great. That said, see: 

https://github.com/bazelbuild/rules_scala/pull/15/files#diff-09498dbadf45966909850dc8a47ebb13R9

bazel's `maven_jar` expects the artifact in one big string.

We saw massive pains at Twitter with pants when we had `_2.9.2` baked into artifact names rather than just one function we could update. Want to make sure the same does not happen here.